src/app_service/service/settings/service.nim

```diff
@@ -442,4 +442,14 @@ method autoMessageEnabled*(self: Service): bool =
 method getWakuBloomFilterMode*(self: Service): bool =
   return self.settings.wakuBloomFilterMode
 
+method areTestNetworksEnabled*(self: Service): bool =
+  return self.settings.testNetworksEnabled
+
+method toggleTestNetworksEnabled*(self: Service): bool =
```

Comment:
this part is again debatable right, like we could call the settings service from the wallet_account.service instead of calling it from the module. We can check if the flow for different services to interact on the cpp side. just putting it here, cause this what I was mentioning on the architecture call.

Reply:
Isn' t it what I am doing (see wallet account service):
```
method toggleTestNetworksEnabled*(self: Service) = 
  discard self.settings_service.toggleTestNetworksEnabled()
  self.tokenService.init()
  self.refreshBalances()
  self.events.emit(SIGNAL_WALLET_ACCOUNT_NETWORK_ENABLED_UPDATED, NetwordkEnabledToggled())
```

copy/pasting code for reference